PowerPointLabs/PowerPointLabs/NarrationsLab/NotesToAudio.cs

```diff
@@ -69,7 +73,11 @@ public static List<string[]> EmbedSelectedSlideNotes()
             List<string[]> audioList = new List<string[]>();
 
             List<PowerPointSlide> slides = PowerPointCurrentPresentationInfo.SelectedSlides.ToList();
-
+            if (NotesToAudio.IsAzureVoiceSelected && UserAccount.GetInstance().IsEmpty())
+            {
+                MessageBox.Show("Invalid user account. Please log in again.");
+                throw new Exception();
```

Comment:
Will there be more contextual information added to the exception?

Reply:
Edited!